avaframe/in4Region/splitInputsCfg.ini

```diff
@@ -0,0 +1,9 @@
+### Config File - This file contains the main settings for splitInputs
+## Copy to  local_splitInputsCfg.ini and set your parameters
+# This file is part of Avaframe.
+
+[GENERAL]
+#if DEM shall be split
```

Comment:
Does it make sense to not split the DEM? Is there a case where this is not wanted?